benefits/eligibility/templates/eligibility/start--calfresh.html

```diff
@@ -16,8 +16,8 @@ <h1>{% translate "You selected a CalFresh Cardholder transit benefit." %}</h1>
 {% endblock media-item %}
 
 {% block call-to-action %}
```

Comment:
Noting this for the #2540 ticket:

All of these can be `{% block call-to-action-button %}` instead of overriding all of `{% call-to-action %}`.  `{% call-to-action %}` only needs to exist for the landing page/agency landing page.